internal/app/i18n/bundle.en.go

```diff
@@ -0,0 +1,68 @@
+package i18n
+
+var en = map[Message]string{
+	CannotDetermineHomeDir:            "Home directory cannot be determined.",
+	CannotDetermineProjectDir:         "Cannot determine the project directory for project %s.\n",
+	ErrorInitLogFile:                  "Failed to initialize log file: %s.\n",
+	VersionInfo:                       "Ktor CLI version %s.\n",
+	LogHint:                           "For more details, see the log: %s.\n",
+	ProjectCreated:                    "Project \"%s\" created at %s.\n",
+	JDKDetectedJavaHome:               "JDK detected in JAVA_HOME=%s.\n",
+	JdkDetected:                       "Detected JDK %s.\n",
+	JdkFoundLocally:                   "Local JDK found at %s.\n",
+	JdkDownloaded:                     "JDK downloaded to %s.\n",
+	JdkVerificationFailed:             "JDK verification failed. Retrying...",
+	GenServerError:                    "Error connecting to the generation server. Please try again later.\n",
+	GenServerTimeoutError:             "Connection to the generation server timed out. Please try again later.\n",
+	NetworkError:                      "Network error while connecting to the generation server. Check your Internet connection and try again.\n",
+	InternalError:                     "Internal error. To get help, report the issue on YouTrack: https://youtrack.jetbrains.com/newIssue?project=ktor.\n",
+	ProjectDirExist:                   "The project directory %s already exists.\n",
+	NoPermsCreateProjectDir:           "Insufficient permissions to create project directory %s.\n",
+	ProjectExtractError:               "Failed to extract downloaded archive to the directory %s.\n",
+	JdkExtractError:                   "Failed to extract downloaded JDK to the directory %s.\n",
+	DirAlreadyExist:                   "The directory already exists.",
+	UnableExtractJdk:                  "Failed to extract downloaded JDK.\n",
+	UnableDownloadJdk:                 "Failed to download JDK %s for %s %s\n",
+	JdkServerError:                    "Error connecting to the JDK server. Please try again later.\n",
+	JdkServerDownloadError:            "Error downloading from the JDK server. Please try again later.\n",
+	ChecksumVerificationFailed:        "Checksum verification failed for the downloaded JDK.",
+	UnableMakeFileExec:                "Failed to make %s executable.\n",
+	UnexpectedError:                   "An unexpected error occurred.\n",
```

Comment:
This isn't a very helpful message. It would be nice to provide some pointers how to solve it.

Reply:
Then it's good!